src/utils/prettify.ts

```diff
@@ -1,24 +1,29 @@
-import path from "path";
 import fs from "fs";
-import shell from "shelljs";
+import fg from "fast-glob";
 import log from "./logging";
+import * as prettier from "prettier";
 
-const prettify = (glob: string) => {
-  const prettierExecutable = path.join(
-    process.cwd(),
-    "node_modules",
-    ".bin",
-    "prettier"
-  );
+const prettify = async (globPattern: string) => {
+  const configFilePath = await prettier.resolveConfigFile();
+  const config = await prettier.resolveConfig(configFilePath || "");
 
-  if (fs.existsSync(prettierExecutable)) {
-    log.info(`Formatting ${glob} files with Prettier`);
-    shell.exec(
-      [prettierExecutable, "--loglevel silent", "--write", `"${glob}"`].join(
-        " "
-      )
-    );
-  }
+  fg.glob(globPattern)
+    .then(files => {
+      files.forEach(async file => {
+        const data = fs.readFileSync(file, "utf-8");
+
+        const pretty = await prettier.format(data, {
+          ...(config || {}),
+          parser: "typescript",
+        });
+
+        fs.writeFileSync(file, pretty);
+        log.success(`Prettified ${file}`);
+      });
+    })
+    .catch(err => {
+      console.log(err);
+    });
```

Comment:
As we are not handling this error, maybe we should let it crash instead. Then at least we know the job failed and have the full stack trace. What do you think?